docs/reference/migration/migrate_8_4.asciidoc

```diff
@@ -20,33 +20,3 @@ coming::[8.4.0]
 There are no breaking changes in {es} 8.4.
 // end::notable-breaking-changes[]
 
-
```

Comment:
I don't think that this information should be removed. The [migration page](https://www.elastic.co/guide/en/elasticsearch/reference/current/migrating-8.4.html) for 8.4 contains deprecation information that relates to the minor version release, and shouldn't be removed for a patch release.

Reply:
Agreed. The info for 8.4 should remain as-is.

Reply:
Reverted the deletion.